test/RetrievesHelper.test.ts

```diff
@@ -0,0 +1,141 @@
+import { getELMFixture } from './helpers/testHelpers';
+import { findRetrieves } from '../src/helpers/RetrievesHelper';
+import { DataTypeQuery } from '../src/types/Calculator';
+
+const simpleQueryELM = getELMFixture('elm/queries/SimpleQueries.json');
+const simpleQueryELMDependency = getELMFixture('elm/queries/SimpleQueriesDependency.json');
+
+const EXPECTED_VS_RETRIEVE_RESULTS: DataTypeQuery[] = [
```

Comment:
Not critical at the moment, but is there a  way to build these off of our bae expected results? Maybe in the future, it would make sense to pull expected results into a separate file that we can essentially pull in as test fixtures?